scheduler/bin/gcloud-make-for-integration.sh

```diff
@@ -0,0 +1,56 @@
+##
+## Configure a kubernetes cluster for running pool-based integration tests and running pools in general.
+##
+##
```

Comment:
Can you add a usage comment, e.g.:

https://github.com/twosigma/Cook/blob/master/scheduler/bin/run-docker.sh#L3-L6

Reply:
Great idea, it also lets me clean up some other test further down.